doc/developer-documentation/forgae-cli/deploy.md

```diff
@@ -0,0 +1,30 @@
+# forgae deploy
+
+## Run deploy script
+
+```
+forgae deploy
+```
+
+The **deploy** command help developers run their deploy script aeternity
```

Comment:
apparently, are some previous mistakes here  this can be changed to : The deploy command helps developers run their deployment scripts and thus deploy their aeternity projects.